plugins/database/src/db_plugin.cpp

```diff
@@ -14824,414 +15121,217 @@ irods::error db_mod_ticket_op(
+
                 if ( logSQL != 0 ) {
                     rodsLog( LOG_SQL, "chlModTicket SQL 14" );
                 }
-                status =  cmlExecuteNoAnswerSql(
-                              "delete from R_TICKET_ALLOWED_HOSTS where ticket_id=? and host=?",
-                              &icss );
-                if ( status == CAT_SUCCESS_BUT_WITH_NO_INFO ) {
-                    return CODE( status );
-                }
-                if ( status != 0 ) {
-                    rodsLog( LOG_NOTICE,
-                             "chlModTicket cmlExecuteNoAnswerSql delete host failure %d",
-                             status );
-                    return ERROR( status, "delete failed" );
-                }
-                status = cmlAudit3( AU_MOD_TICKET, ticketIdStr,
-                                    _ctx.comm()->clientUser.userName,
-                                    _ctx.comm()->clientUser.rodsZone, "remove host",
-                                    &icss );
-                if ( status != 0 ) {
-                    return ERROR( status, "cmlAudit3 failed" );
-                }
-                status =  cmlExecuteNoAnswerSql( "commit", &icss );
-                if ( status < 0 ) {
-                    return ERROR( status, "commit failed" );
-                }
-                else {
-                    return SUCCESS();
-                }
 
-            }
+                return execute_sql("delete from R_TICKET_ALLOWED_HOSTS where ticket_id=? and host=?");
+            } // host
+
             if ( strcmp( _arg4, "user" ) == 0 ) {
                 status = icatGetTicketUserId( _ctx.prop_map(), _arg5, user2IdStr );
                 if ( status != 0 ) {
                     return ERROR( status, "icatGetTicketUserId failed" );
                 }
+
                 i = 0;
                 cllBindVars[i++] = ticketIdStr;
                 cllBindVars[i++] = _arg5;
                 cllBindVarCount = i;
+
                 if ( logSQL != 0 ) {
                     rodsLog( LOG_SQL, "chlModTicket SQL 15" );
                 }
-                status =  cmlExecuteNoAnswerSql(
-                              "delete from R_TICKET_ALLOWED_USERS where ticket_id=? and user_name=?",
-                              &icss );
-                if ( status == CAT_SUCCESS_BUT_WITH_NO_INFO ) {
-                    return CODE( status );
-                }
-                if ( status != 0 ) {
-                    rodsLog( LOG_NOTICE,
-                             "chlModTicket cmlExecuteNoAnswerSql delete user failure %d",
-                             status );
-                    return ERROR( status, "delete failed" );
-                }
-                status = cmlAudit3( AU_MOD_TICKET, ticketIdStr,
-                                    _ctx.comm()->clientUser.userName,
-                                    _ctx.comm()->clientUser.rodsZone, "remove user",
-                                    &icss );
```

Comment:
should these calls be restored?

they do not exist in the main-line/4.3 branch.

Reply:
I think this has to do with audit trails, which were removed over there. Not sure if it should be removed for 4.2.x, but as far as I know, removal of audit trails wasn't supposed to happen til 4.3.x.

Reply:
that is correct... https://github.com/irods/irods/issues/4337

Reply:
I'll restore these calls.

Reply:
done.